content/en/docs/instrumentation/java/automatic/server-config.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Application server configuration
+linkTitle: App server config
```

Comment:
Can we shorten this a bit more (given that this page is quite nested in the left nav)?

```suggestion
linkTitle: Server config
```